src/analysis/analytic/linear.lean

```diff
@@ -50,4 +51,47 @@ protected theorem has_fpower_series_at (f : E →L[𝕜] F) (x : E) :
 protected theorem analytic_at (f : E →L[𝕜] F) (x : E) : analytic_at 𝕜 f x :=
 (f.has_fpower_series_at x).analytic_at
 
+/-- Reinterpret a bilinear map `f : E →L[𝕜] F →L[𝕜] G` as a multilinear map
+`(E × F) [×2]→L[𝕜] G`. This multilinear map is the second term in the formal
```

Comment:
This is not a completely natural reinterpretation, as the most natural one would be a multilinear map on the modules `(E, F)`, not on `(E × F, E × F)`. Maybe add some details to the docstring, saying that `uncurry_bilinear f ((u, v), (u', v')) = f (u, v')` (is that right?)